pkg/kubelet/prober/prober.go

```diff
@@ -137,13 +149,15 @@ func (pb *prober) runProbe(ctx context.Context, probeType probeType, p *v1.Probe
 	timeout := time.Duration(p.TimeoutSeconds) * time.Second
 	switch {
 	case p.Exec != nil:
-		klog.V(4).InfoS("Exec-Probe runProbe", "pod", klog.KObj(pod), "containerName", container.Name, "execCommand", p.Exec.Command)
+		klog.V(3).InfoS("Exec-Probe runProbe", "pod", klog.KObj(pod), "containerName", container.Name, "execCommand", p.Exec.Command)
```

Comment:
@jralmaraz Can you explain the idea behind changing log level here and below? From the first look it's not related to the PR that claims to improve handling of the `probe.Warning` result.

Reply:
I think I got them mixed up, more of a mistake on my end. Moving it back